plugin/src/main/kotlin/kotlinx/coroutines/CoroutineExceptionHandlerImpl.kt

```diff
@@ -0,0 +1,7 @@
+package kotlinx.coroutines
+
+import kotlin.coroutines.CoroutineContext
+
+fun handleCoroutineExceptionImpl(context: CoroutineContext, exception: Throwable) {
+    exception.printStackTrace();
```

Comment:
@MrMineO5, does this actually work? I was planning on fixing this issue once in for all by wrapping all calls in `GlobalScope.launch` with try/catch.

Reply:
It prints the actual stacktrace of the exception rather than a NoClassDefFoundError if that's what you mean

Reply:
Good to know